packages/kbn-es-query/src/filters/build_filters/types.ts

```diff
@@ -82,6 +82,12 @@ export type Query = {
   language: string;
 };
 
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+export type AggregateQuery = {
```

Comment:
This is a nice idea but I would like to investigate it further when we add the esql. I just added it there for reference and it is what we had agreed on the RFC. (If I change it now, I will have some type failures so I would prefer to look into it later!

Reply:
This is a nice idea but I would like to investigate it further when we add the esql support. I use it here as a reference as it is the type that was proposed on the RFC. If I change it now it will create many types failures so I would prefer to investigate it later!

Reply:
I will look into it further

Reply:
Thanx for that, this is how it was suggested on the RFC but I missed it somehow 😓 Addressed!